tests/util/setup-smoke-tests.ts

```diff
@@ -18,6 +18,14 @@ export function describeSmokeSuite(
   options: SmokeTestOptions,
   cb: (context: SmokeTestContext) => void
 ) {
+  if (!options.wssUrl) {
+    throw Error(`Missing wssUrl parameter (use WSS_URL=... npm run smoke-test)`);
+  }
+
+  if (!options.relayWssUrl) {
```

Comment:
Some tests don't require the relay

Reply:
Hmm, is there a way to know which ones do or do not require the relay? I didn't even know about that.

Reply:
I think this `before()` fn would need to be modified to support optional relay connection anyway...

Reply:
Sure, I adapted the before() to optionally call `ApiPromise.create` for `relayWssUrl`. It'll use the real instance if the parameter is provided or throw an error if any test attempts to use an `unimplemented` proxy object.


```typescript
// test.ts
before("get data from relay chain", async function () {
  await context.relayApi.query.system.digest();
});
```
```
  0 passing (938ms)
  1 failing

  1) Verify balances consistency
       "before all" hook: get data from relay chain for "should have matching deposit/reserved":
     Error: unimplemented! Requires `RELAY_WSS_URL` parameter
```